common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -43,6 +43,9 @@ public class InventoryEmeraldCountFeature extends UserFeature {
     @Config
     public boolean showContainerEmeraldCount = true;
 
+    @Config
+    public boolean showZerosInEmeraldText = false;
```

Comment:
Generalize this config value for the rendered version too (at least in name and i18n)